app/lib/controllers/todo_controller.dart

```diff
@@ -306,6 +308,26 @@ class ToDoController extends GetxController {
     );
   }
 
+  // Mock data for subscribed users
+  List<SubscriberModel> listSubscribers = <SubscriberModel>[
+    SubscriberModel('', 'Okon Invincible', false),
+    SubscriberModel('', 'Floym Dore', false),
+    SubscriberModel('', 'Raveena Tondon', false),
+    SubscriberModel('', 'Karishma Kapoor', false),
+    SubscriberModel('', 'Hema Malini', false),
+    SubscriberModel('', 'John Doe', false),
+  ];
+
+  void handleCheckClick(int position) {
+    var subscribeModel = listSubscribers[position];
+    if (subscribeModel.isSelected) {
+      subscribeModel.isSelected = false;
+    } else {
+      subscribeModel.isSelected = true;
+    }
+    update();
```

Comment:
Please use this call with widget ids as it'll only update the widget that really needs and also helps tracking the stacktrace.